ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -330,6 +330,7 @@ literals:
 
 ## Added on 10/25/2018
 - compressor_speed_percentage_command
+- oil_compressor_pressure_sensor
```

Comment:
oil pump? dont think you can compress liquid oil.

Reply:
Actually it is a compressor oil pressure sensor . Since oil is a descriptor and compressor a component so the above name for field is used.

Reply:
suggest using `oil_pressure_sensor` and implementing it in an oil monitoring abstract type

Reply:
Removed the compressor oil pressure sensor as not able to model as per the requirement.